include/deal.II/lac/vector_operations_internal.h

```diff
@@ -909,6 +919,10 @@ namespace internal
         // vec_size == 0
         outer_results[0] = ResultType();
 
+        // the variable serves two purposes: (i) number of chunks (each 32 indices)
+        // there are for the given size, op result are stored in outer_results[0,n_chunks).
```

Comment:
rephrase: number of chunks (a 32 indices) for the given size; all results are stored in outer_results[0,n_chunks)